src/Strategy/Conda.hs

```diff
@@ -0,0 +1,58 @@
+module Strategy.Conda (
```

Comment:
Please run ormolu on this file.

Reply:
Is there a more automated way of doing this? I feel like this could be a pre-commit hook or something

Reply:
If you want to set up your own pre-commit hook, you can.  See [the ormolu usage docs](https://github.com/tweag/ormolu#usage) for invocation help.

However, it's worth noting that I do not think we should **enforce** git hooks in any repository.  Anything that needs to be checked should be done in CI.  We can do that using `ormolu --mode check FILES`.

Reply:
There is also an ormolu github action [here](https://github.com/marketplace/actions/ormolu-action)